packages/forms/src/widgets/TogglePasswordWidget/TogglePasswordWidget.scss

```diff
@@ -0,0 +1,23 @@
+.btn-group {
+    width: 100% !important;
```

Comment:
really needed ?

Reply:
if i don't force it the field is reduce at 200px and i don't find why.